src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -31,7 +31,8 @@ namespace Microsoft.Terminal.Settings.Model
     enum BellStyle
     {
         Audible = 0x1,
-        Visual = 0x2,
+        Window = 0x2,

```

Comment:
We might want to slap a big "DANGER! IF YOU TOUCH THIS, YOU MUST UPDATE src/cascadia/TerminalSettingsEditor/Profiles.xaml"